content/docs/rfcs/13/README.md

```diff
@@ -9,29 +9,29 @@ contributors:
   - Oskar Thorén <oskar@status.im>
 ---
 
-This specification explains the Waku `WakuStore` protocol which enables querying of messages received through relay protocol and stored by other nodes. 
+This specification explains the Waku `WAKU2-STORE` protocol which enables querying of messages received through relay protocol and stored by other nodes. 
```

Comment:
I'd prepend as `13/WAKU-STORE` to make it clearer that it is referring to that unique ID

Reply:
sure, addressed.